manage4.js

```diff
@@ -14,6 +14,84 @@ function loadcfg(ns, config) {
     }
     return cfg
 }
+const PURCHASE_NAME = "DO_NOT_TOUCH_SERVER"

```

Comment:
Please change this to be configurable or shorter so that tools like `scan` don't have terrible output.

Suggestion: `autoserver`